aiohttp/web.py

```diff
@@ -221,6 +221,22 @@ def middlewares(self):
         return self._middlewares
 
     def make_handler(self, **kwargs):
+        try:
+            debug = kwargs['debug']
```

Comment:
`helpers._sentinel` is a marker which could be used even if `None` is proper value.
Not necessary for boolean parameters though.